.github/workflows/build.yml

```diff
@@ -0,0 +1,97 @@
+name: alfresco Extension Inspector Build Workflow
+
+on:
+  push:
+    branches:
+      - master
+      - 'fix/**'
+      - 'support/**'
+      - 'feature/**'
+  pull_request:
+    types: [opened, synchronize]
+    branches:
+      - master
+      - 'fix/**'
+      - 'support/**'
+      - 'feature/**'
```

Comment:
keep in mind that `branches` for `pull_request` have a very different meaning than `push`:
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-your-workflow-based-on-the-head-or-base-branch-of-a-pull-request

you probably only want `master` here, and that would trigger a workflow for every PR against master

Reply:
you are also getting duplicated workflows due this (one triggered by push and another triggered by pull_request) for the very same commit

Reply:
I've added a suggestion to try to fix this